### PR TITLE
Add English version

### PR DESCRIPTION
**Pull request by renardpal**
Connected to openfisca/openfisca.org#68

I created every json file for french & english page. It has no "key" so far. I don't quite know where/how to translate the components (hero / footer / header)

### CODE REVIEW

File: static/language/en/resources.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Resources
+
+System: France
+45 contributors on the project
```
Comment (antoningarrone) [inline static/language/en/resources.json]: Je mettrais juste "45 contributors" ça peut créer une confusion entre les used case et les system si le mot de projet est utilisé pour les 2 et personnellement je préférè "projects" à "showcase". C'est valable aussi pour le site français.

File: static/language/en/showcase/fastoche.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Fastoch€
+
+Back
+
+Fastoch€ is a financial coach.
+
+Produced by PayBoost, a company specialized in online payment.
```
Comment (antoningarrone) [inline static/language/en/showcase/fastoche.json]:
```suggestion
Developed by PayBoost, a company specialized in online payment.
```

File: static/language/en/showcase/fastoche.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Fastoch€ is a mobile app that works as a financial coach. It aims at helping users to deal with their monthly bugdget. It also informs them on elegibility to social benefits and how to apply for those benefits.
+
+Fastoch€ was launched in 2018.
+
+Visit Fastoch€
+
+Project attributes
+uses the web API
+questions the French system
+runs a simulation on individuals
```
Comment (antoningarrone) [inline static/language/en/showcase/fastoche.json]: Je mettrais "run simulations on individuals"

File: static/language/en/showcase/index.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Showcase
+
+Mes Aides
+
+Mes Aides informs French citizens on their eligibility to 32 social benefits. 
+
+LexImpact
+
+This on going project aims at creating a tool to simulate the impact of a reform on the French government’s budget and on a population’s standard of living.
```
Comment (antoningarrone) [inline static/language/en/showcase/index.json]:
```suggestion
 An ongoing project that aims to create a simple tool to simulate the impact of socio-fiscal reforms on the French government’s budget and on the population’s standard of living.
```

File: static/language/en/showcase/index.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+Tax IPP
+
+This simulation model matches administrative data and survey data so they can be sent into OpenFisca, to analyse the effects of a tax and benefit system on households.
```
Comment (antoningarrone) [inline static/language/en/showcase/index.json]:
```suggestion
This simulation model matches administrative data and survey data so they can be plug to OpenFisca, to analyse the impacts of a tax and benefit system on households. 
```

File: static/language/en/showcase/index.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+This on going project aims at creating a tool to simulate the impact of a reform on the French government’s budget and on a population’s standard of living.
+
+Tax IPP
+
+This simulation model matches administrative data and survey data so they can be sent into OpenFisca, to analyse the effects of a tax and benefit system on households. 
+
+Rates Rebates
+
+This prototype digital service makes it easier for low-income homeowners to understand their entitlement and apply for a "Rates Rebate" subsidy.
```
Comment (antoningarrone) [inline static/language/en/showcase/index.json]:
```suggestion
This prototype made by New Zealand's government innovation team tests the automatic payment of a social benefit.
```

File: static/language/en/showcase/index.json [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+Fastoch€
+
+This mobile app is a social and money coach. It allows users to easily deal with the monthly budget and to get information on social benefits.
```
Comment (antoningarrone) [inline static/language/en/showcase/index.json]:
```suggestion
This mobile app is a social and money coach. It allows users to easily deal with their monthly budget and to get information on social benefits. 
```

File: static/language/en/showcase/lexImpact.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
+LexImpact
+
+Back
+
+Simulate the impact of tax reforms on citizens. 
+
+Powered by Etalab and part of the EIG program (General Interest Entrepreneur), the team works with admninistrator within the French National Assembly.
+
+This project aims at helping administrators and deputees to simulate the impact of reforms on the government budget before its vote.
```
Comment (antoningarrone) [inline static/language/en/showcase/lexImpact.json]:
```suggestion
This project aims at helping administrators and deputees of the French parliements to simulate the impact of socio-fiscal amendment before its vote. 
```

File: static/language/en/showcase/mesAides.json [hunk 9]
```diff
@@ -0,0 +1,19 @@
+Mes Aides
+
+Back
+
+Mes Aides is a social benefits simulator.
+
+Built by BetaGouv, the French government innovation team.
+Mes Aides informs French citizens on their eligibility to national and local social benefits. In less than 7 minutes, the service gives an answer on more than 30 benefits.  
+
+Also uses some custom-made extensions build by the team: city local benefits, tool to trace the calculation details and navigate through it.
```
Comment (antoningarrone) [inline static/language/en/showcase/mesAides.json]:
```suggestion
Mes Aides also designed some custom-made extensions such as city local benefits or tools to trace the calculation details and navigate through it. 
```

File: static/language/en/showcase/ratesRebates.json [hunk 10]
```diff
@@ -0,0 +1,20 @@
+Rates rebates
+
+Back
+
+Prototyping automated service for social benefits.
+
+Built by the Service Innovation Lab team (New Zealand), this prototype makes it easy for senior users to apply only for the Rates rebates subsidy.
```
Comment (antoningarrone) [inline static/language/en/showcase/ratesRebates.json]:
```suggestion
Built by the Service Innovation Lab team (New Zealand), this prototype makes it easy for senior users to apply for the Rates rebates subsidy.
```

File: static/language/en/showcase/taxIpp.json [hunk 11]
```diff
@@ -0,0 +1,19 @@
+TaxIpp
+
+Back
+
+TaxIPP is a micro simulation model.
```
Comment (antoningarrone) [inline static/language/en/showcase/taxIpp.json]:
```suggestion
TaxIPP is a microsimulation model.
```

File: static/language/en/showcase/taxIpp.json [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+This model mixes survey data and administrative data to get a representative sample of households. The data is then injected into OpenFisca to analyse the impact of a tax & benefit system on households.
+
+TAXIPP is used by the IPP (The Institute for Public Policy) which animate public debate since 2012.
+
+Among other subjects, TaxIPP analyses the French government finance law every year.
```
Comment (antoningarrone) [inline static/language/en/showcase/taxIpp.json]:
```suggestion
Among other subjects, TaxIPP analyses the French government Finance Law every year. 
```

File: static/language/en/showcase/lexImpact.json (hunk 8 above)
Comment (renardpal) [reply static/language/en/showcase/lexImpact.json]: Assembly already mentioned on the previous sentence

File: static/language/en/showcase/index.json (hunk 6 above)
Comment (renardpal) [reply static/language/en/showcase/index.json]: official text from the lab website

**Comment by sandcha** [comment]
@renardpal Voir cette [documentation](https://react.i18next.com/legacy-v9/step-by-step-guide) pour `react.i18next.com` (v9, avant dernière).

**Comment by renardpal** [comment]
stopped where I couldn't go further:
- how to deal with url in the text to be translated?
- how to deal with components (doesn't seem to work to translate the props)
- how to make it work outside the index...

**Comment by sandcha** [comment]
Rebase à la dernière version de `master` en cours.

**Comment by renardpal** [comment]
I'm unable to: 

- [ ] fix the "orivacy policy" page

- [ ] include properly the links included in paragraph of text as a translation key inside a key (page contribute)

- [ ] do the toggle for language swap (active language / inactive language)
Capture d’écran 2019-06-03 à 11 48 05

File: public/locales/fr/index.json [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+    "feature1-list-b":"Codez votre propre système",
+    "feature1-list-c":"Améliorez le code d’un système existant",
+    "feature1-cta":"Voir les systèmes socio-fiscaux",
+    "feature2-list-a":"Importez des données",
+    "feature2-list-a":"Calculez une situation ou écrivez et simulez une réforme",
+    "feature2-list-b":"Codez des extensions",
+    "feature2-cta":"Tester l'API web",
+    
+    "title-community":"Une communauté open source internationale",
+    "content-community":"OpenFisca a des contributeurs partout dans le monde ! Rejoignez une communauté d’économistes, de chercheurs, de développeurs et d’experts des droits sociaux qui s’engagent dans la transformation de la loi en code informatique !",
```
Comment (Morendil) [inline public/locales/fr/index.json]: ça pourrait être plus inclusif :)

File: public/locales/fr/community.json [hunk 14]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Communauté",
+    "title-who": "Qui sommes-nous ?",
+    "content-who": "OpenFisca est une communauté Open Source soutenue par Beta Gouv & Etalab, deux équipes d’innovation au sein du gouvernement français. En 2019, OpenFisca est utilisée par 7 pays dans le monde (France, Espagne, Italie, Nouvelle Zélande, Sénégal & Tunisie). Plus de 70 personnes ont rejoint cette aventure. Ils contribuent, s’entraident et participent chaque jour à améliorer le code d’OpenFisca.",
```
Comment (Morendil) [inline public/locales/fr/community.json]: BetaGouv en un seul mot

File: public/locales/en/showcase/taxIpp.json [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+    "title": "TaxIpp",
+    "back":"back",
+    "baseline" : "TaxIPP is a microsimulation model.",
+    "desc1":"This model mixes survey data and administrative data to get a representative sample of households. The data is then injected into OpenFisca to analyse the impact of a tax & benefit system on households.",
+    "desc2":"TAXIPP is used by the IPP (The Institute for Public Policy) which animate public debate since 2012.",
+    "desc3":"Among other subjects, TaxIPP analyses the French government Finance Law every year.",
+    "textlink":"Visit TaxIPP",
+    "car":"Project attributes",
+    "api":"uses the Python API",
+    "country":"questions the French system",
```
Comment (Morendil) [inline public/locales/en/showcase/taxIpp.json]: questions -> queries

File: public/locales/fr/index.json (hunk 13 above)
Comment (sandcha) [reply public/locales/fr/index.json]: @Morendil Comment l'écris-tu ? 😅 
> Toolbox : point médian `·`

**Pull request merged**